source/postprocess/dynamic_topography.cc

```diff
@@ -124,6 +124,14 @@ namespace aspect
               double dynamic_topography_x_volume = 0;
               double volume = 0;
 
+              // check which way gravity points
```

Comment:
please mention in a half sentence this might become important in backward advection models